website/templates/emails/confirm.txt.mako

```diff
@@ -1,6 +1,8 @@
 Hello ${user.fullname},
 
-Welcome to the Open Science Framework. Please verify your email address by visiting this link:
+This email address has been added to an account on the Open Science Framework.
```

Comment:
I think it will be better if after "an account" you can tell which account it has been added to.